pgcli/pgexecute.py

```diff
@@ -354,9 +339,28 @@ def _columns(self, kinds=('r', 'v', 'm')):
                 'm' - materialized view
         :return: list of (schema_name, relation_name, column_name, column_type) tuples
         """
+        column_order = {'table_order': 'att.attnum', 'alphabetic': '3'}[
+            self.config['main']['column_order']]
+
+        columns_query = '''
+            SELECT  nsp.nspname schema_name,
+                    cls.relname table_name,
+                    att.attname column_name,
+                    typ.typname type_name
+            FROM    pg_catalog.pg_attribute att
```

Comment:
what if we store attnum in the column metadata object, and then have the *-expansion code in `get_column_matches` sort by whatever's appropriate there? table-order vs alphabetical order might come up later in other completions so instead of fixing it at the level of `pgexecute.columns` we should fix it at the site of the suggestion